tests/context.py

```diff
@@ -14,7 +14,8 @@
     "max_encode": 33521664,
     "debug": False,
     "cleanup": False,
-    "epoch": None
+    "epoch": None,
+    "stable_repos": False
```

Comment:
Some test is still failing in CI: https://github.com/asmap/kartograf/actions/runs/15609591484/job/43967070558?pr=82